config/dev

```diff
@@ -1,5 +1,11 @@
+instanceid = dev
+
 include Makefile
 
+export base_url = /${instanceid}
```

Comment:
In ui dev config, I've set api_url to the make serve one : http://localhost:6543
if you want to use dev wsgi instance in ui dev config set api_url to http://vhost/instanceid/
for the main instance the base url is / so it reachable with http://vhost/